models/tridimensional/docking_validation/batch_dock.sh

```diff
@@ -74,16 +87,25 @@ if [ ! -z $set_partners ]; then
     echo "Adding '--set-partners $set_partners' argument to dock_variants"
     opt_dock_variants_args+=" --set_partners $set_partners"
 fi
+if [ ! -z $pam64 ]; then
+    echo "Passing '--pam64' argument to dock_variants"
+    opt_dock_variants_args+=" --pam64"
+    pam_end=63
+fi
+if [ ! -z $pam_tool]; then
+    echo "Adding '--alt_tool' argument to dock_variants"
+    echo "Adding '--alt_tool' argument to results_csv"
+    opt_dock_variants_args+=" --alt_tool"
+    opt_results_csv_args=" --alt_tool"
+fi
 
 # don't write to a directory that already exists
 results_dir=$RESULTS/batch/$label
 if [ -d $results_dir ]; then
-	echo "Directory $results_dir already exists, specify a new label"
-	exit 1
+    echo "Directory $results_dir already exists, specify a new label"
+    exit 1
 fi
 
-pam_start=0
-pam_end=255
 range=$(($pam_end-$pam_start+1))
```

Comment:
$pam_start and $pam_end were move to line 34 & 35. This was done so $pam_end can be modified in the $pam64 check on lines 90-93, so that 64 variant tests have the proper end.